components/Cards/FlatCard/__tests__/FlatCard.test.js

```diff
@@ -18,14 +17,25 @@ describe('FlatCard', () => {
     ),
   };
 
+  const imageProps = {
+    imageSource: `${s3}headshots/david_molina.jpg`,
+    imageAlt: "David Molina's face",
+    ...requiredProps,
+  };
+
   it('should render with required props', () => {
     createSnapshotTest(<FlatCard {...requiredProps} />);
   });
 
+  it('should render with image props', () => {
+    createSnapshotTest(<FlatCard {...imageProps} />);
```

Comment:
We still need to pass required props here

Reply:
`imageProps` object has `requiredProps` properties by using spread operator right now (line 23). If that's hard to read I can refactor to `<FlatCard {...imageProps} {...requiredProps} />` on line 31.

Reply:
Missed that! Thanks